### PR TITLE
index page with showing only the password

### PR DESCRIPTION
**Pull request by StavAlush27**
i added a split to the data

### CODE REVIEW

File: lab2/lab2/index.py [hunk 1]
```diff
@@ -8,8 +8,7 @@ def printL(req):
 
 @handler("af","POST")
 def answerpost(req):
-    data=post(req)
-    
-    return 200,{"Content-type":"text/html"},data #return_code,headers,code of the page(here it is html)
+    data=(str(post(req))).split('&')
+    return 200,{"Content-type":"text/html"},data[0] #return_code,headers,code of the page(here it is html)
```
Comment (yotam180) [inline lab2/lab2/index.py:12]: Operator spacing (e.g. `200, {"....": "..."}, data[0]`)

File: lab2/lab2/index.py [hunk 2]
```diff
@@ -8,8 +8,7 @@ def printL(req):
 
 @handler("af","POST")
 def answerpost(req):
-    data=post(req)
-    
-    return 200,{"Content-type":"text/html"},data #return_code,headers,code of the page(here it is html)
+    data=(str(post(req))).split('&')
```
Comment (yotam180) [inline lab2/lab2/index.py:11]: Again — operator spacing.
And also remove the unnecessary parentheses wrapping `str(post(req))`

Also, do you really want to treat your `data` as `bytes` and not as a `str`?

**Review by yotam180 — CHANGES_REQUESTED** [review]
Left some comments for code quality assurance.

**Comment by nofearOnline** [comment]
please leave it for now and write the report as mentioned in the issue

**Comment by nofearOnline** [comment]
it has already done so I close the issue

**Pull request closed without merging**